x86_64 Assembly.tmbundle/Syntaxes/Nasm Assembly.sublime-syntax

```diff
@@ -544,7 +544,7 @@ contexts:
         - include: strings
         - include: line-ending
         - include: preprocessor-macro-indirection
-    - match: '^\s*((%)(?:assign|i?deftok|strcat|strlen|substr|pathsearch|push|pop|repl|line|clear))\b'
+    - match: '^\s*((%)(?:assign|i?deftok|strcat|strlen|substr|pathsearch|push|pop|rotate|repl|line|clear))\b'
```

Comment:
`%rotate` is only defined inside a multi-line macro, and it is already covered (see line 863)